1.2.c

```diff
@@ -0,0 +1,72 @@
+#include <stdio.h>
+#include <stdlib.h>
+
+
+// Task: ����������� ������ ����������>
+
+size_t n = 0;
+long* arr = NULL;
+
+void sortType()
+{
+	size_t i;
+	printf("Array size: ");
+	scanf_s("%u", &n);
+
+	arr = (long*)calloc(n, sizeof(long));
+
+	for (i = 0; i < n; i++) 
+	{
+		printf("Array[%d] =  ", i);
+
+		scanf_s("%d", &(arr[i] ));

```

Comment:
```suggestion
		scanf_s("%d", &arr[i]);
```
Лишние скобки, проблемы с форматированием кода (лишний пробел).
(-0.5 балла)